doc/modules/pipeline.rst

```diff
@@ -1,276 +1,8 @@
-.. _combining_estimators:
+.. raw:: html
 
-===============================================
-Pipeline and FeatureUnion: combining estimators
-===============================================
+    <meta http-equiv="refresh" content="1; url=./compose.html" />
+    <script>
+      window.location.href = "./compose.html";
```

Comment:
Just curious, what is the point of both setting `window.location.href` and using `http-equiv="refresh"` solution? I thought only the latter would work.